ERCS/eip-agrocrypto-quantum.md

```diff
@@ -0,0 +1,350 @@
+---
+
+eip: TBD
```

Comment:
```suggestion
eip: 8040
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.